src/mito2/src/worker/handle_write.rs

```diff
@@ -294,8 +311,89 @@ impl<S> RegionWorkerLoop<S> {
                 sender_req.sender,
             );
         }
+    }
 
-        region_ctxs
+    /// Processes bulk insert requests.
+    fn process_bulk_requests(
+        &mut self,
+        region_ctxs: &mut HashMap<RegionId, RegionWriteCtx>,
+        requests: &mut Vec<SenderBulkRequest>,
+    ) {
```

Comment:
Mostly the same as `process_write_requests()`? I think we need to refactor this to avoid duplicate code.